gatsbyjs/src/components/Grid.js

```diff
@@ -1,12 +1,11 @@
 import React from "react";
```

Comment:
I think we can remove this import since `React` is not being used in this file.

Reply:
@fgiuliani It threw me an error

```
'React' must be in scope when using JSX  react/react-in-jsx-scope
```

All the components files have JSX and I put it back.

Reply:
@schabibi1 good point!